packages/oidc-js/src/utils/sign-in.ts

```diff
@@ -92,8 +92,12 @@ export function hasAuthorizationCode(requestParams: ConfigInterface | WebWorkerC
  */
 export function getAuthorizationCode(requestParams?: ConfigInterface | WebWorkerConfigInterface): string {
     if (!requestParams || !isWebWorkerConfig(requestParams)) {
-        if (new URL(window.location.href).searchParams.get(AUTHORIZATION_CODE)) {
-            return new URL(window.location.href).searchParams.get(AUTHORIZATION_CODE);
+        if (!requestParams || requestParams.responseMode !== "form_post") {
```

Comment:
Shall we covert `ResponseModeTypes` from type to an enum and use it here like `ResponseModeTypes.FORM_POST`. WDYT?

Reply:
Yes, we need to do that. However, doing this will necessitate changes to identity apps. So, I have created an issue to track this: https://github.com/asgardio/asgardio-js-oidc-sdk/issues/51